CHANGELOG.md

```diff
@@ -12,13 +12,12 @@ All notable changes to AET will be documented in this file.
 
 ## Version 3.0.1
 
-
+- [PR-394](https://github.com/Cognifide/aet/pull/394) ([#386](https://github.com/Cognifide/aet/issues/386)) Added scroll modifier
```

Comment:
Please move this line to "Unreleased" section

Reply:
Done